src/Microsoft.Dnx.Tooling/Restore/RestoreCommand.cs

```diff
@@ -799,7 +825,10 @@ private Task<LockFile> ReadLockFile(string projectLockFilePath)
             }
 
             // Record all libraries used
-            foreach (var item in graphItems.OrderBy(x => x.Match.Library, new LibraryComparer()))
+            var items = graphItems
```

Comment:
Instead of using linq why not just do the loop once and  exclude things in there?